### PR TITLE
fix: Mobile layout: hide closed positions disappears #3344

### PR DESCRIPTION
**Pull request by ghgoodreau**
This change adds the 'Hide closed positions' button to the mobile Pool page view. As referenced in #3344 , without this, the pool page can accumulate closed positions and the request can even time out with a large number of positions. We shouldn't limit user functionality due to mobile view whenever possible.

By default, the button is identical to the font size of the 'Your positions'. This supports widths up to 340px.
image

The smallest mobile view that I typically support is 320px (iPhone 5). With this, the bar overflows and doesn't look great.
image

With a font size of 14px on 'Hide closed positions', we are able to fit the 'Your positions' + 'Hide closed positions' in one line on screen widths as small as 320px.
image

### CODE REVIEW

**Comment by willhennessy** [comment]
thank you! You're right the overflow case looks a bit funny.

this week our designer [specced](https://github.com/Uniswap/interface/issues/1828#issuecomment-1132197701) a new on/off toggle for this and other toggles throughout the app. It can help here.

I think the right overflow behavior is draw the toggle element on a new line below "Your Positions"
Screen Shot 2022-05-22 at 1 33 15 PM
 
would you be up for that?

**Comment by ghgoodreau** [comment]
@willhennessy Yeah, I'm up for that! I'll take a look at the spec and try to update the PR within the next couple days. Thanks!

**Comment by ghgoodreau** [comment]
See attached video demo and breakpoint screenshots. Also, I named the component 'SimpleToggle', but can change it to whatever naming preferences you have.

https://user-images.githubusercontent.com/25194960/169713335-632333f5-ef8b-445f-823f-17ea6cc576c8.mp4

image

image

image

**Comment by ghgoodreau** [comment]
@willhennessy Let me know what you think about my changes!

**Comment by willhennessy** [comment]
this looks great! summoning @infredible for design review and I'll find an engineer to review the code

**Comment by infredible** [comment]
Looks great! Only piece of feedback would be to align the toggle to the right of the container when it wraps.

**Comment by ghgoodreau** [comment]
> Looks great! Only piece of feedback would be to align the toggle to the right of the container when it wraps.

Thanks @infredible ! Just to be clear, do you mean the toggle itself aligned to the right, while the 'Show closed positions' text stays aligned on the left? Similar to this? 

image

**Comment by infredible** [comment]
Yes @ghgoodreau! That's exactly what I meant

**Comment by ghgoodreau** [comment]
XS Mobile breakpoint style updated as per design review and improved some component naming
image

Think its ready for code review @vm

**Review by lynnshaoyu — APPROVED** [review]
Looks ready to merge to me. Thanks for making this toggle @ghgoodreau !

**Comment by ghgoodreau** [comment]
Anytime, thank you for taking a look @lynnshaoyu ! Do I just wait for someone with write access to merge now?

**Comment by lynnshaoyu** [comment]
Hey @ghgoodreau it's merged now! Thanks again 😃🙏

**Pull request merged**
